data/schema/v1/Decision_Point-1-0-1.schema.json

```diff
@@ -41,23 +47,32 @@
 		},
 		"namespace": {
 		    "type": "string",
-		    "description": "Namespace (a short, unique string): For example, \"ssvc\" or \"cvss\" to indicate the source of the decision point"
+		    "description": "Namespace (a short, unique string): For example, \"ssvc\" or \"cvss\" to indicate the source of the decision point. See SSVC Documentation for details.",
+		    "pattern": "^[a-z0-9-]{3,4}[a-z0-9/\\.-]*$",
```

Comment:
In your last approach, one registered value was `cvssv4`. Did you skip the version intentionally?

Reply:
Not intentional - but that is a good point. We use the namespace always as `cvss` and never as `cvssv4`  to identify distinct CVSS versions. Discussion needed internally adding @ahouseholder  for awareness.
 
./src/ssvc/decision_points/cvss/base.py 

https://github.com/CERTCC/SSVC/blob/3fd228b2cc83f59e94f7a00bf773233c7671e8c7/src/ssvc/decision_points/cvss/base.py#L28

Reply:
At this point, the namespace for any CVSS-derived decision point, regardless of CVSS version, is likely to remain "cvss". We're trying to use the namespace to indicate concepts that are defined by some other authority (e.g., the CVSS SIG in this case) and where our decision points are just intended to reflect their definitions. In other words, we don't "own" the semantics of those decision points, we're just providing them for convenience to SSVC users.

Writing that helps me solidify some thinking about #703 too.